src/panels/config/dashboard/ha-config-navigation.ts

```diff
@@ -68,6 +71,17 @@ class HaConfigNavigation extends LitElement {
     ha-navigation-list {
       --navigation-list-item-title-font-size: var(--ha-font-size-l);
     }
+    /* Accessibility */
+    .visually-hidden {
+      position: absolute;
+      overflow: hidden;
+      clip: rect(0 0 0 0);
+      height: 1px;
+      width: 1px;
+      margin: -1px;
+      padding: 0;
+      border: 0;
```

Comment:
I bet just `top: 999999px, left: 999999px` would work just as well but anyway. I guess this will be more robust against 16k displays 😆

Reply:
It seemed this was the best practice that I could find recommended. I saw we have a similar style here as well:

https://github.com/home-assistant/frontend/blob/33d5cecc8541bf8b454c324aee3d66b5836d7ffc/src/components/ha-big-number.ts#L98